lib/src/over_react_redux/over_react_redux.dart

```diff
@@ -32,6 +32,15 @@ import 'package:redux/redux.dart';
 
 part 'over_react_redux.over_react.g.dart';
 
+/// This class is present:
+///
+/// 1. to allow for consumers which have used the --backwards-compat flag with over_react_codemod to statically analyze:
+///     <https://github.com/Workiva/over_react_codemod/blob/71e5713ec6c256ddaf7c616ff9d6d26d77bb8f25/README.md#dart-1-to-dart-2-codemod>
+/// 2. to provide forwards-compatibility and allow this to be mixed into mixin-based component props
+abstract class $ConnectPropsMixin {
+  static const PropsMeta meta = _$metaForConnectPropsMixin;
```

Comment:
I'm pretty sure this should be deprecated

Reply:
Should the class itself also be deprecated? (Same question for all the other `$MixinName` instances in this PR)

Reply:
Good points, yup, they should be

Reply:
Actually, for deprecating the classes themselves, I wonder if we should wait until after the bulk of the boilerplate transition has occurred so that everyone doesn't get deprecation warnings for legacy boilerplate and erroneously remove them. Thoughts?

Reply:
I have deprecated all of the constants, though